stdm_config/views.py

```diff
@@ -189,25 +190,25 @@ def EntityLookupSummaries(profile, entity):
 
 def LooukupSummary(child_entity, ers):
```

Comment:
Function names should be in lowercase and separated by underscore if required. See PEP8 guidelines [here](https://peps.python.org/pep-0008/#descriptive-naming-styles). Kindly also review other parts of the code to ensure conformance.

Reply:
Yes I'm addressing the code style in a separate branch which I'll open a  PR for